src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -987,8 +971,8 @@
     <comment>Header for a control to toggle classic CRT display effects, which gives the terminal a retro look.</comment>
   </data>
   <data name="Profile_RetroTerminalEffect.HelpText" xml:space="preserve">
-    <value>When enabled, enables retro terminal effects such as glowing text and scan lines.</value>
-    <comment>A description for what the "retro terminal effects" setting does. Presented near "Profile_RetroTerminalEffect".</comment>
+    <value>Show "retro" terminal effects such as glowing text and scan lines.</value>

```

Comment:
I think "retro-style" would be better than quoting the word "retro". It also seems like this translates better into other languages (judging by Google Translate).